### PR TITLE
bump mongo to 6.0.7

### PR DESCRIPTION
**Pull request by brianlball**
bump mongoid to 7.4.3

### CODE REVIEW

**Comment by brianlball** [comment]
https://github.com/NREL/OpenStudio-server/issues/708

File: server/Gemfile [hunk 1]
```diff
@@ -69,7 +69,7 @@ gem 'sassc', '~> 2.4.0'
 
 gem 'openstudio-workflow', '= 2.3.1'
 #gem 'openstudio-analysis', :github => 'NREL/OpenStudio-analysis-gem', :ref => 'develop'
-gem 'openstudio-analysis', '= 1.3.2'
+gem 'openstudio-analysis', '= 1.3.4'
 
 # Remove urbanopt cli gems for minor for releases as they use different versions of ext gems
 gem 'urbanopt-cli', '= 0.9.2'
```
Comment (kflemin) [inline server/Gemfile:75]: this could be 0.9.3

**Review by kflemin — APPROVED** [review]

**Comment by brianlball** [comment]
Note:  if you are using docker, you will need docker engine >= 20.10 to use Mongo >= 6.0.4 [issue](https://github.com/docker-library/mongo/issues/606)

**Review by nllong — APPROVED** [review]
big bump! nice!

**Pull request merged**